sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -0,0 +1,1188 @@
+## API Report File for "@azure/storage-file-datalake"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { AbortSignalLike } from "@azure/core-http";
+import { AbortSignalLike as AbortSignalLike_2 } from "@azure/abort-controller";
```

Comment:
Why are two `AbortSignalLike` imported?

Reply:
Different imports in different places! Let's import from the authoritative source only (@azure/abort-controller).

Reply:
Sure!